rmgweb/database/models.py

```diff
@@ -33,18 +33,20 @@
 from rmgweb.database.tools import database
 
 
-def getSolventList():
+def get_solvent_list():
     """
     Return list of solvent molecules for initializing solvation search form.
     If any of the Mintz parameters are None, that solvent is not shown in the list since it will cause error.
```

Comment:
Everything looks really good! Just a really small nitpick, this comment should be deleted because it is no longer true with these changes.
Other than that, I approve of your changes.

Reply:
Thanks! That change is made with this commit 4a2ca4c. If you approve this, I will squash the fixup commits and force-push.

Reply:
Looks good!